### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rmatambo8**
Visual changes i think would be helpful for the readme.

nothing that i consider essential though

### CODE REVIEW

**Comment by ghost** [comment]
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**gatsby-starter-landing-page**

:tada: Your build was successful! [See the Deploy preview here.](https://landingpagestarter-rmatambo8patch1.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/ab141368-6f33-4c23-82ed-5531dcc907f8/sites/7cf5e89e-66b7-49b9-b5b6-472b5d946224/builds/fba95424-6aa7-4f5d-9355-b662868e534c/details)

:clock1: Build time: **14s**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :green_heart: 99   |
| Accessibility |   :green_heart: 100   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 88   |

:link: [View full report](https://build-fba95424-6aa7-4f5d-9355-b662868e534c.gtsb.io/reports/lighthouse/index.html)

File: gatsby-theme-landing-page/README.md [hunk 1]
```diff
@@ -1,68 +1,70 @@
...
-(Optional) If you only want to import the content model and not the demo data, use the command: `yarn import:contentful:model`
+    - (Optional) If you only want to import the content model and not the demo data, use the command: `yarn import:contentful:model`
 
-Your Contentful space will now contain the content model used by the starter, along with demo content that demonstrates how to use the various content types and landing page components.
+    Your Contentful space will now contain the content model used by the starter, along with demo content that demonstrates how to use the various content types and landing page components.
 
 ## Content model
 
-![content model diagram](https://user-images.githubusercontent.com/1227297/145318779-4b2cca42-4ae9-407c-b8f6-db934628c51c.png)
+    ![content model diagram](https://user-images.githubusercontent.com/1227297/145318779-4b2cca42-4ae9-407c-b8f6-db934628c51c.png)
```
Comment (thinkybeast) [inline gatsby-theme-landing-page/README.md]: This image isn't appearing properly (seems to be wrapped in a code block) 
image

**Review by thinkybeast — APPROVED** [review]
Looks good outside of the image formatting issue. Thanks for making these updates!

File: gatsby-theme-landing-page/README.md (hunk 1 above)
Comment (jxnblk) [reply gatsby-theme-landing-page/README.md]:
```suggestion
![content model diagram](https://user-images.githubusercontent.com/1227297/145318779-4b2cca42-4ae9-407c-b8f6-db934628c51c.png)
```

I think this should not be indented. 4 space indent == code block

File: gatsby-theme-landing-page/README.md [hunk 2]
```diff
@@ -143,7 +147,7 @@ To completely ignore the default CSS custom properties defined in the theme, add
 }
 ```
 
-## Section components
+### Section components
```
Comment (jxnblk) [inline gatsby-theme-landing-page/README.md:150]: These sections don't all neatly fit under "Customization" so I think we'd need to adjust the content a bit to change all the H2s to H3s

File: gatsby-theme-landing-page/README.md [hunk 3]
```diff
@@ -1,68 +1,70 @@
...
 
-Your Contentful space will now contain the content model used by the starter, along with demo content that demonstrates how to use the various content types and landing page components.
+    Your Contentful space will now contain the content model used by the starter, along with demo content that demonstrates how to use the various content types and landing page components.
 
 ## Content model
 
 ![content model diagram](https://user-images.githubusercontent.com/1227297/145318779-4b2cca42-4ae9-407c-b8f6-db934628c51c.png)
 
-Each landing page is represented by a hierarchical content model:
+    Each landing page is represented by a hierarchical content model:
```
Comment (jxnblk) [inline gatsby-theme-landing-page/README.md]:
```suggestion
Each landing page is represented by a hierarchical content model:
```

This also shouldn't be indented

File: gatsby-theme-landing-page/README.md [hunk 4]
```diff
@@ -234,7 +261,7 @@ query ($id: String!) {
 }
 ```
 
-## Customizing buttons, links, and other components
+### buttons, links, and other components
```
Comment (jxnblk) [inline gatsby-theme-landing-page/README.md]:
```suggestion
### Buttons, links, and other components
```

**Comment by jxnblk** [comment]
Generally LGTM, pending the code block formatting changes

File: gatsby-theme-landing-page/README.md (hunk 2 above)
Comment (jxnblk) [reply gatsby-theme-landing-page/README.md:150]: Not a blocker BTW; noting this more to follow up on

File: gatsby-theme-landing-page/README.md (hunk 1 above)
Comment (rmatambo8) [reply gatsby-theme-landing-page/README.md]:
```suggestion
![content model diagram](https://user-images.githubusercontent.com/1227297/145318779-4b2cca42-4ae9-407c-b8f6-db934628c51c.png)
```

**Pull request merged**